module/VuFind/src/VuFind/Navigation/AbstractMenu.php

```diff
@@ -56,7 +56,7 @@ public function __construct(protected array $config)
      */
     public function getMenu(): array
     {
-        $menu = $this->config ?: $this->getDefaultMenu();
+        $menu = $this->config ?: $this->getDefaultMenuConfig();
```

Comment:
Shouldn't `$this->getDefaultMenuConfig()` become `static::getDefaultMenuConfig()` (here, and in other places where called) if we want to use a static method here?